src/pass.h

```diff
@@ -104,22 +118,22 @@ class Pass : public WasmWalker {
 // e.g. through PassRunner::getLast
 
 // Handles names in a module, in particular adding names without duplicates
-class NameManager : public Pass {
+class NameManager : public WalkerPass<WasmWalker<NameManager, void> > {
```

Comment:
C++11 made the extra space in `> >` not necessary.

I just use `clang-format` on all the code I write so there's never any style discussion ;-)